docs/Migrating.md

```diff
@@ -1,5 +1,11 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.5 to v0.6
+### Important
+* Brains are now Scriptable Objects instead of Monobehaviors. This will
```

Comment:
Maybe give some recommendation about the best way to move a project that is using a lot of the old brains to use the new brains. Especially since the old functionality is removed.

Reply:
I added a paragraph.